pages/Graph/Overview.md

```diff
@@ -6,8 +6,12 @@ menu: Graph
 
 # Graph
 
-This specification is intended to describe the social network graph and how we represent it in the protocol.
-This section should also cover how our graph representation could be applied to other networks, i.e. the friendship model vs follower model.
+This specification describes the social network graph and how it is represented in the protocol. 
+
+In this context a [social graph](https://en.wikipedia.org/wiki/Social_graph) means a graph that represents social relations between entities.
+
+The DSNP graph represents users as [DSNP Ids](/Identity/Overview). Each instance of a social network graph is made up of a DSNP Id following other DSNP Ids.
+When the term "user" is referenced below we are referring to a DSNP Id.
```

Comment:
(Not blocking) I think we've been trying to avoid the word "we" in the spec. Maybe we could just say something like, `The term "user" below refers to a DSNP User Id.`